chef/cookbooks/apt/.delivery/project.toml

```diff
@@ -0,0 +1 @@
+remote_file = "https://raw.githubusercontent.com/chef-cookbooks/community_cookbook_tools/master/delivery/project.toml"
```

Comment:
Per overall review comments this file should not be checked into this repository directly.